paimon-core/src/test/java/org/apache/paimon/mergetree/ChangelogMergeTreeRewriterTest.java

```diff
@@ -192,6 +192,9 @@ public void testRewriteSuccess(boolean rewriteChangelog) throws Exception {
 
     private KeyValueFileWriterFactory createWriterFactory(
             Path path, RowType keyType, RowType valueType) {
+        Options options = new Options();
+        options.set(CoreOptions.DATA_FILE_THIN_MODE, false);
+
```

Comment:
Default value of `CoreOptions.DATA_FILE_THIN_MODE` is `false`, why do we need this change? Even if the default value changes to `true`, will it break the test?